atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TodoEteTest.java

```diff
@@ -34,4 +39,38 @@ public void shouldBeAbleToWriteAndListTodos() {
         todoClient.addTodo(TODO);
         assertThat(todoClient.getTodoList(), hasItem(TODO));
     }
+
+    @Test
+    public void shouldSweepStreamIndices() {
+        TodoResource todoClient = EteSetup.createClientToSingleNode(TodoResource.class);
+
+        storeFiveStreams(todoClient, 4321);
+
+        SweepResults firstSweep = todoClient.sweepSnapshotIndices();
+        assertThat(firstSweep.getCellTsPairsExamined(), equalTo(9L));
+        assertThat(firstSweep.getStaleValuesDeleted(), equalTo(4L));
+
+        SweepResults valueSweep = todoClient.sweepSnapshotValues();
+        assertThat(valueSweep.getCellTsPairsExamined(), equalTo(9L));
+        assertThat(valueSweep.getStaleValuesDeleted(), equalTo(4L));
+
+        storeFiveStreams(todoClient, 7654321);
+
+        SweepResults secondSweep = todoClient.sweepSnapshotIndices();
+        assertThat(secondSweep.getCellTsPairsExamined(), equalTo(15L));
```

Comment:
Generally some explanation as to why these numbers are what is expected would be welcome